src/open_inwoner/plans/models.py

```diff
@@ -115,6 +116,15 @@ class Plan(models.Model):
         blank=True,
         help_text=_("How do you intend to achieve this goal?"),
     )
+    note = ckeditor_fields.RichTextField(
+        max_length=1000,
+        default=_(""),
+        blank=True,
+        verbose_name=_("Edit note message"),
```

Comment:
For the admin the `verbose_name` can be useful; for the citizen I'm not so sure. We already have the title from the page and the help text below the input field, also from the HTML. Suggestions:

- Change the text for the beheerder: `"Edit note"` instead of `"Edit note message"`
- Hide this from the editor for the citizen. To this end, update the `PlanNoteForm` as follows:

```
class PlanNoteForm(forms.ModelForm):
    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)
        self.fields["note"].label = ""

    class Meta:
        [your code...]

```

Of course, if there's a design/style reason to keep the label, then keep it.

Reply:
We have no design, and no reason.